pattern-distributed/confs/sp-worker/conf/deployment.yaml

```diff
@@ -238,80 +210,66 @@ data.agent.config:
   reporting:
     console:
-      - # The name for the Console Reporter
-        name: Console
-
+        # The name for the Console Reporter
+    - name: Console
         # Enable Console Reporter
-        enabled: false
-
+      enabled: false
         # Polling Period in seconds.
         # This is the period for polling metrics from the metric registry and printing in the console
-        pollingPeriod: 5
-
+      pollingPeriod: 5
 wso2.metrics.jdbc:
   # Data Source Configurations for JDBC Reporters
   dataSource:
-    # Default Data Source Configuration
-    - &JDBC01
       # JNDI name of the data source to be used by the JDBC Reporter.
       # This data source should be defined in a *-datasources.xml file in conf/datasources directory.
-      dataSourceName: java:comp/env/jdbc/WSO2MetricsDB
+  - &JDBC01
+    dataSourceName: java:comp/env/jdbc/WSO2MetricsDB
       # Schedule regular deletion of metrics data older than a set number of days.
       # It is recommended that you enable this job to ensure your metrics tables do not get extremely large.
       # Deleting data older than seven days should be sufficient.
-      scheduledCleanup:
+    scheduledCleanup:
         # Enable scheduled cleanup to delete Metrics data in the database.
-        enabled: false
-
+      enabled: false
         # The scheduled job will cleanup all data older than the specified days
-        daysToKeep: 7
-
+      daysToKeep: 3
         # This is the period for each cleanup operation in seconds.
-        scheduledCleanupPeriod: 86400
-
+      scheduledCleanupPeriod: 86400
   # The JDBC Reporter is in the Metrics JDBC Core feature
   reporting:
     # The JDBC Reporter configurations will be ignored if the Metrics JDBC Core feature is not available in runtime
     jdbc:
-      - # The name for the JDBC Reporter
-        name: JDBC
-
+        # The name for the JDBC Reporter
+    - name: JDBC
         # Enable JDBC Reporter
-        enabled: false
-
+      enabled: 'false'
         # Source of Metrics, which will be used to identify each metric in database -->
         # Commented to use the hostname by default
         # source: Carbon
-
         # Alias referring to the Data Source configuration
-        dataSource: *JDBC01
-
+      dataSource: *JDBC01
         # Polling Period in seconds.
         # This is the period for polling metrics from the metric registry and updating the database with the values
-        pollingPeriod: 60
-
+      pollingPeriod: 60
   # Deployment configuration parameters
 wso2.artifact.deployment:
     # Scheduler update interval
   updateInterval: 5
-
   # Periodic Persistence Configuration
 state.persistence:
   enabled: true
   intervalInMin: 1
   revisionsToKeep: 2
   persistenceStore: org.wso2.carbon.stream.processor.core.persistence.DBPersistenceStore
   config:
-    datasource: SP_WORKER_STATE_DB   # A datasource with this name should be defined in wso2.datasources namespace
+    location: siddhi-app-persistence
```

Comment:
In DBPersistenceStore, we don't have a location parameter. Hence, we can remove it

Reply:
will do